swagger/prime.yaml

```
@@ -859,6 +859,10 @@ definitions:
         readOnly: true
       agents:
         $ref: '#/definitions/MTOAgents'
+      mtoServiceItems:
+        items:
+          $ref: '#/definitions/MTOServiceItem'
+        type: array
```

Ideally, this would look something like: 
```suggestion
        mtoServiceItems:
            $ref: '#/definitions/MTOServiceItem'
```
After trying this several times, it looks like `go-swagger` had problems handling the definition this way. Very strange, especially considering `MTOAgents` had [no issue being defined this way](https://github.com/transcom/mymove/blob/17d2b4ae8f49ce1bd36f9e6432b8f6902d62762a/swagger/prime.yaml#L860). I think there is a bug in swagger's validation library that is mishandling definitions with multiple types, like [MTOServiceItemModelType](https://github.com/transcom/mymove/blob/17d2b4ae8f49ce1bd36f9e6432b8f6902d62762a/swagger/prime.yaml#L963)